show/main.py

```diff
@@ -1917,6 +1917,210 @@ def mmu():
     click.echo(proc.stdout.read())
 
 
+#
```

Comment:
since this is a new group, please separate into a different file. check show/vlan.py for example.